shell/assets/translations/en-us.yaml

```diff
@@ -1279,6 +1279,8 @@ cluster:
         macAddress: Mac Address
         macFormat: 'Invalid MAC address format.'
       vGpus:
+        warnings:
+          minimumAllocatable: It's highly recommended to select a vGPU with a number of allocatable devices greater then the number of nodes (Machine Count) to avoid "un-schedulable" errors after cluster updates.
```

Comment:
Similar to feedback on the 2.10 & 2.9 related PRs:
- `then` should be *than*
- `vGPU` should be used with that capitalization for consistency